views/myCv.hbs

```diff
@@ -0,0 +1,42 @@
+
+    <div class="formPosition thumbnail">  
+      <div class="cvHeader">
+        <div>
+          <img src={{actual.avatar_url}} class="small_Image imags" alt="sentayhu's pictures here">
+        </div>
+        <div>  
+          <br>
+          <h3 class="persInfo_Format">{{actual.name}}</h3>
+          <p>{{actual.location}}<br>
+          Tel. (07575) 84216<br>
+          E-mail: {{actual.email}}</p>
+        </div>
+      </div>
+    
+    <h1 class="persInfo_Format ">Personal Statment</h1>
+      <p>With extensive experience in computer software development and implementation, I adapt well to various situations and enjoy learning new skills to keep up-to-date with new software and programming techniques. Previous employment has taught me the importance of completing work to a very high standard at all times and has honed excellent management skills and an acute sense of professional responsibility. Pro-active, reliable and conscientious, I have a proven ability to function effectively, both within a group working towards a common goal and unsupervised, using my own initiative. I feel I would be a useful addition to your workforce.</p>
+    <h1 class="persInfo_Format">Skils</h1>
+      <ul>
+        <li>Analytical thinking and planning</li>
+        <li>Strong verbal, formal and inter-personal communication skills</li>
+        <li>Problem analysis, use of judgement and ability to solve problems efficiently.</li>
+        <li>Self motivation, initiative, high level of energy.</li>
+        <li>Tolerance and flexibility to different situations.</li>
+        <li>Adaptability and ability to work under extreme pressure.</li>
+        <li>System and process modelling technique skills.</li>
+        <li>Computer Hardware and Software maintenance,</li>
+      </ul> 
+    <h1 class="persInfo_Format">Eemployement History</h1>
+      <ul>
+        <li>Feb 2016 - Aug 2016   Voluntary Database Developer   Kensington Community Learning Centre (Liverpool, UK)
+        </li>
+        <li>Feb 2016 - Aug 2016   Voluntary Database Developer   Kensington Community Learning Centre (Liverpool, UK)</li>
+      </ul>
+    <h1 class="persInfo_Format">Qualification and Training</h1>
+      <h2 class="qual_Format"></h2>
+      <h2 class="qual_Format"></h2>
+    <h1 class="persInfo_Format">Reference</h1>
```

Comment:
Almost none of these `h1`s should be `h1`s, they're almost all subheadings and should be `h2` or `h3`